src/main/java/mistermusik/logic/Command.java

```diff
@@ -774,11 +774,30 @@ public void instrumentManagement(InstrumentList instruments, UI ui) {
         }
     }
 
-    //@@author
+    //@@author Dng132FEI
+    
+    public static boolean isNumeric(String str) { 
+	    try {  
+	        Integer.parseInt(str);  
+	        return true;
+	    } catch(NumberFormatException e){  
+	        return false;  
+	    }  
+	}
+    
     private void remindEvents(EventList events, UI ui) {
-        ui.printReminder(events);
+    	if (continuation == " ") {
```

Comment:
Can change `continuation == ""` to `continuation.isEmpty()`.

Reply:
> I remember we have a NumberFormatException exception, is that same as your isNumeric method?
> and can the command "reminder" without a space aft it work now?

NumberFormatException is a java api exception that happens when you try to parse an integer that isn't an integer. can still manually throw it though. In the exception handling for user inputs i mostly used it whenever it wasn't a valid number entered.